application/src/main/java/org/thingsboard/server/service/resource/TbResourceService.java

```diff
@@ -35,9 +36,9 @@ public interface TbResourceService extends SimpleTbEntityService<TbResource> {
 
     TbResourceInfo findResourceInfoById(TenantId tenantId, TbResourceId resourceId);
 
-    PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TenantId tenantId, PageLink pageLink);
```

Comment:
it is better to use simple variable name that is more readable (just ```filter``` instead of ```tbResourceInfoFilter```)